lambda_function/src/file_processor/file_processor.py

```diff
@@ -241,6 +278,12 @@ def _process_file(self) -> None:
                             file_key=new_file_key,
                         )
 
+                        if self.tracker:
+                            # Track file in CDF
+                            self.tracker.track(
+                                Path(calibration.process_file(file_path)[0])
```

Comment:
Do you need a second call to `process_file` here?

Reply:
Good catch! Fixed it in the latest commit